src/Sylius/Behat/Context/Api/Shop/OrderContext.php

```diff
@@ -257,6 +259,21 @@ public function theShouldHaveUnitPriceDiscountedFor(ProductInterface $product, i
         Assert::same(-$discount, $amount);
     }
 
+    /**
+     * @Then I should have chosen :paymentMethod payment method
+     */
+    public function iShouldHaveChosenPaymentMethodForMyOrder(PaymentMethod $paymentMethod): void
+    {
+        $payment = $this->iriConverter->getItemFromIri(
+            $this->responseChecker->getValue($this->client->show($this->sharedStorage->get('cart_token')), 'payments')[0]
+        );
```

Comment:
I would be for getting payment method by an API call instead of translating payment to an item